app/services/users/create_mascot_account.rb

```diff
@@ -0,0 +1,29 @@
+module Users
+  class CreateMascotAccount
+    MASCOT_PARAMS = {
+      email: "mascot@forem.com",
+      username: "mascot",
+      name: "Mascot",
+      profile_image: SiteConfig.mascot_image_url,
+      confirmed_at: Time.current,
+      registered_at: Time.current,
+      password: SecureRandom.hex
+    }.freeze
+
+    def self.call
+      new.call
+    end
+
+    def call
+      return if SiteConfig.mascot_user_id
+
+      mascot = User.create(mascot_params)
```

Comment:
```suggestion
      mascot = User.create!(mascot_params)
```
Any reason not to fail hard if for some reason the mascot user isn't created?